packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -29,6 +29,11 @@ import 'theme.dart';
 /// The [onReorder] parameter is required and will be called when a child
 /// widget is dragged to a new position.
 ///
+/// By default, a drag handle is stacked over the center of each item's trailing
+/// edge on desktop platforms only; on mobile platforms, no handle is visible
+/// and a long press anywhere on the item starts a drag. To customize this
+/// behavior check [ReorderableListView.buildDefaultDragHandles].
```

Comment:
```suggestion
/// edge on [TargetPlatformVariant.desktop] platforms. On [TargetPlatformVariant.mobile] platforms, no handle is visible
/// and a long press anywhere on the item starts a drag. To change this
/// behavior use [ReorderableListView.buildDefaultDragHandles].
```

Reply:
While I like this wording, it currently isn't true 😄.  The reorderable list treats `TargetPlatform.fuchsia` as a desktop platform, but the `TargetPlatformVariant` puts it in the `mobile` category. We should fix the reorderable list view to treat it as mobile as well to be consistent.

That said, we should go ahead with this and I will take care of fixing the issue in https://github.com/flutter/flutter/issues/84376.